instrumentation/httpasyncclient/src/main/java/brave/httpasyncclient/TracingHttpAsyncClientBuilder.java

```diff
@@ -91,6 +91,10 @@ public static HttpAsyncClientBuilder create(HttpTracing httpTracing) {
 
   static final class RemoveScope implements HttpRequestInterceptor {
     @Override public void process(HttpRequest request, HttpContext context) {
+      process(context);
+    }
+
+    static void process(HttpContext context) {
```

Comment:
small change.. pull up as static method removeScope. then its use inline doesn't need qualification and is a bit more readable.

Reply:
Makes sense - I've updated the PR